jsapp/scss/components/_kobo.table.scss

```diff
@@ -21,217 +26,329 @@
+      background: transparent;
+      border: 0;
+      padding: 0;
 
-      .-loading-inner {
-        display: block;
-        text-align: center;
-        font-size: 18px;
-        opacity: 0.8;
-        color: $kobo-gray-40;
+      &:hover {
+        color: $kobo-blue;
+      }
 
-        i {
-          margin-right: 12px;
-          vertical-align: middle;
-          font-size: 24px;
-        }
+      i {
+        font-size: 24px;
+        margin: 0 3px;
+        vertical-align: middle;
       }
     }
+  }
+
+  .-pagination {
+    box-shadow: none;
+    border-top: none;
+    padding: 0;
 
-    &.-highlight .rt-tbody .rt-tr:not(.-padRow):hover {
-      background: $kobo-teal-light;
+    .-center {
+      flex: 2;
     }
 
-    &.has-frozen-column {
-      .frozen {
-        order: -1;
-        position: relative;
-        left: 0;
-        background: $kobo-gray-96;
-        height: 36px;
-        transition: left 0.2s;
-        margin-right: 6px;
+    .-btn {
+      color: $kobo-blue;
+      font-size: 12px;
+      line-height: 14px;
+      font-weight: 700;
+      text-transform: uppercase;
+      background: transparent;
 
-        &:not(.rt-status) {
-          z-index: 2;
+      &[disabled] {
+        pointer-events: none;
+      }
 
-          &:hover {
-            z-index: 3;
-          }
-        }
+      .k-icon {
+        vertical-align: top;
       }
 
-      &.-striped .rt-tr.-odd .frozen {
-        background: $kobo-gray-96;
+      &:hover {
+        color: $kobo-blue-dark;
+        background: transparent;
       }
 
-      &.-striped .rt-tr:hover .frozen {
-        background: $kobo-teal-light;
+      &:active {
+        transform: translateY(1px);
       }
+    }
 
-      .rt-th.frozen {
-        height: 34px;
+    .-pageJump {
+      margin-left: 10px;
+      margin-right: 10px;
+
+      input {
+        appearance: none;
       }
     }
+
+    select {
+      cursor: pointer;
+    }
+
+    input,
+    select {
+      border: 1px solid $kobo-gray-92;
+      background: $kobo-white;
+      height: 32px;
+      padding: 5px 7px;
+      font-size: 14px;
+      border-radius: 6px;
+      font-weight: 700;
+    }
+  }
+
+  .pagination-bottom {
+    margin-top: 5px;
+    margin-bottom: 20px;
+  }
+
+  .-loading {
+    background: $kobo-gray-96;
+
+    .-loading-inner {
+      display: block;
+      text-align: center;
+      font-size: 18px;
+      opacity: 0.8;
+      color: $kobo-gray-40;
+
+      i {
+        margin-right: 12px;
+        vertical-align: middle;
+        font-size: 24px;
+      }
+    }
+  }
+
+  &.-highlight .rt-tbody .rt-tr:not(.-padRow):hover {
+    background: $kobo-gray-98;
+  }
+
+  &.has-frozen-column {
+    .frozen {
+      order: -1;
+      position: relative;
+      left: 0;
+      background: $kobo-gray-96;
+      height: 36px;
+      transition: left 0.2s;
+      margin-right: 6px;
+
+      &:not(.rt-status) {
+        z-index: 2;
+
+        &:hover {
+          z-index: 3;
```

Comment:
The z-index seems to be off when hovering over the question name now:
![image](https://user-images.githubusercontent.com/16762675/121826059-c69ea580-cc83-11eb-96f5-ed201bdda1ef.png)

Reply:
Fixed that too :) It was transparent background. Also fixed HXL tags not being displayed